easybuild/easyconfigs/b/BEDTools/BEDTools-2.28.0-foss-2018b.eb

```diff
@@ -0,0 +1,39 @@
+# Author: Maxime Schmitt, University of Luxembourg
+# Author: Adam Huffman, The Francis Crick Institute
+#
+# Based on the work of: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics (SIB)
+# Biozentrum - University of Basel
+
+easyblock = 'MakeCp'
+
+name = 'BEDTools'
+version = '2.28.0'
+
+github_account = 'arq5x'
+homepage = 'https://github.com/%(github_account)s/%(namelower)s2'
+description = """The BEDTools utilities allow one to address common genomics tasks such as finding feature overlaps
+ and computing coverage. The utilities are largely based on four widely-used file formats: BED, GFF/GTF, VCF,
+ and SAM/BAM."""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+
+source_urls = ['https://github.com/arq5x/%(namelower)s2/releases/download/v%(version)s/']
+sources = ['%(namelower)s-%(version)s.tar.gz']
+checksums = ['15af6d10ed28fb3113cd3edce742fd4275f224bc06ecb98d70d869940220bc32']
+
+buildopts = 'CXX="$CXX"'
+
+dependencies = [
+    ('zlib', '1.2.11'),
```

Comment:
Missing XZ and bzip2 as deps

Reply:
@Micket I added the missing dependencies.

Should I also create a PR for existing `BEDTools` easyconfigs where they are also missing?

Reply:
Well, I don't know if it applies to older configs (and I don't really have time to review)